content/en/core/releases/4.5.0.md

```diff
@@ -0,0 +1,100 @@
+
+---
+title: "4.5.0 release notes"
+linkTitle: "4.5.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.5.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+- [#8513](https://github.com/medic/cht-core/issues/8513): Align "Target" components with Material design guidelines
+- [#8541](https://github.com/medic/cht-core/issues/8541): Options with long names look wrong in enketo selects
+- [#8686](https://github.com/medic/cht-core/issues/8686): Fix target UI to match Figma
+
+
+## Highlights
+### Schedules can be created by passing array on start_from
+We can now configure start_from to accept array of fields while configuring schedules. This allows users to pass multiple fields and first existing field will be used to create schedules. Read more about schedules in [documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/schedules/#app_settingsjson-schedules). cht-conf 3.20.0 or higher is required to validate config containing arrays of field in start_from. 
```

Comment:
As long as the cht-conf requirement is documented in the docs I don't think you need to include it here.

```suggestion
It is now possible to configure the `start_from` property to accept array of fields while configuring SMS schedules. This allows users to pass multiple fields and the first existing field will be used to create schedules. Read more about schedules in [the documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/schedules/#app_settingsjson-schedules).
```